### ISSUE TITLE
gitlab (LoadError)

### ISSUE THREAD

**Issue by chraibi** [iv-mexx/gitlab-migrator#2]
I want to migrate a Gitlab-server to another one. Both running on different Linux-machine.

From my laptop (OS X Yosemite 10.10.5) I try to use `gitlab-migrator`

I edited the file `env` as stated in the readme. 

Running 

```
fastlane list_projects
```

fails with the following errors:

```
/System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/rubygems/core_ext/kernel_require.rb:126:in `require': cannot load such file -- gitlab (LoadError)
        from /System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/rubygems/core_ext/kernel_require.rb:126:in `require'
        from /Users/chraibi/gitlab-migrator/fastlane/actions/gitlab_create_project.rb:6:in `<module:Actions>'
        from /Users/chraibi/gitlab-migrator/fastlane/actions/gitlab_create_project.rb:2:in `<module:Fastlane>'
        from /Users/chraibi/gitlab-migrator/fastlane/actions/gitlab_create_project.rb:1:in `<top (required)>'
        from /System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/rubygems/core_ext/kernel_require.rb:126:in `require'
        from /System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/rubygems/core_ext/kernel_require.rb:126:in `require'
        from /Library/Ruby/Gems/2.0.0/gems/fastlane-1.48.0/lib/fastlane/actions/actions_helper.rb:83:in `block in load_external_actions'
        from /Library/Ruby/Gems/2.0.0/gems/fastlane-1.48.0/lib/fastlane/actions/actions_helper.rb:82:in `each'
        from /Library/Ruby/Gems/2.0.0/gems/fastlane-1.48.0/lib/fastlane/actions/actions_helper.rb:82:in `load_external_actions'
        from /Library/Ruby/Gems/2.0.0/gems/fastlane-1.48.0/lib/fastlane.rb:29:in `<module:Fastlane>'
        from /Library/Ruby/Gems/2.0.0/gems/fastlane-1.48.0/lib/fastlane.rb:20:in `<top (required)>'
        from /System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/rubygems/core_ext/kernel_require.rb:73:in `require'
        from /System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/rubygems/core_ext/kernel_require.rb:73:in `require'
        from /Library/Ruby/Gems/2.0.0/gems/fastlane-1.48.0/bin/fastlane:6:in `<top (required)>'
        from /usr/bin/fastlane:23:in `load'
        from /usr/bin/fastlane:23:in `<main>'

```

My environment:

`ruby -v; bundle -v; gem -v`

```
ruby 2.0.0p481 (2014-05-08 revision 45883) [universal.x86_64-darwin14]
Bundler version 1.11.2
2.0.14
```

Before starting I run `bundle install` 
Here is the output I get:

```
Rubygems 2.0.14 is not threadsafe, so your gems will be installed one at a time. Upgrade to Rubygems 2.1.0 or higher to enable parallel gem installation.
Using i18n 0.7.0
Using json 1.8.3
Using minitest 5.8.1
Using thread_safe 0.3.5
Using addressable 2.3.8
Using extlib 0.9.16
Using multi_json 1.11.2
Using babosa 1.0.2
Using colored 1.2
Using highline 1.7.8
Using security 0.1.3
Using excon 0.45.4
Using plist 3.1.0
Using rubyzip 1.1.7
Using multipart-post 2.0.0
Using terminal-table 1.4.5
Using multi_xml 0.5.5
Using coderay 1.1.0
Using method_source 0.8.2
Using slop 3.6.0
Using claide 0.9.1
Using unf_ext 0.0.7.1
Using dotenv 2.0.2
Using mini_magick 4.0.4
Using rouge 1.10.1
Using net-ssh 3.0.1
Using pbxplorer 1.0.0
Using mime-types 2.6.2
Using netrc 0.7.8
Using slack-notifier 1.4.0
Using jwt 1.5.2
Using little-plugger 1.1.4
Using memoist 0.12.0
Using retriable 1.4.1
Using terminal-notifier 1.6.3
Using bundler 1.11.2
Using tzinfo 1.2.2
Using fastimage 1.6.8
Using launchy 2.4.3
Using autoparse 0.3.3
Using commander 4.3.5
Using credentials_manager 0.10.0
Using faraday 0.9.2
Using httparty 0.13.4
Using pry 0.10.3
Using unf 0.1.4
Using xcpretty 0.2.1
Using net-sftp 2.1.2
Using logging 2.0.0
Using activesupport 4.2.4
Using sentry-raven 0.15.2
Using faraday_middleware 0.10.0
Using signet 0.6.1
Using gitlab 3.4.0 from git://github.com/NARKOZ/gitlab.git (at master@ef41dfa)
Using domain_name 0.5.25
Using xcpretty-travis-formatter 0.0.4
Using xcodeproj 0.28.2
Using fastlane_core 0.26.3
Using spaceship 0.14.2
Using krausefx-shenzhen 0.14.6
Using googleauth 0.4.2
Using http-cookie 1.0.2
Using gym 1.1.5
Using scan 0.3.1
Using snapshot 1.1.1
Using cert 1.2.3
Using deliver 1.5.1
Using pem 1.0.1
Using pilot 1.0.1
Using produce 1.0.0
Using sigh 1.1.3
Using xcode-install 1.0.1
Using google-api-client 0.8.6
Using rest-client 1.8.0
Using frameit 2.3.0
Using supply 0.2.1
Using fastlane 1.40.0
Bundle complete! 2 Gemfile dependencies, 77 gems now installed.
Use `bundle show [gemname]` to see where a bundled gem is installed.

```

Any idea what went wrong here? In `gitlab_create_project.rb:L6` there is a 
`require 'gitlab'`,  But `gitlab` is already installed. See:

```
Using gitlab 3.4.0 from git://github.com/NARKOZ/gitlab.git (at master@ef41dfa)
```

Edit:
I solved the problem by explicitly installing gitlab:

```
sudo gem install gitlab
```

I thought this is already done by `bundle install`, is it not?

**Comment by iv-mexx** [comment]
I'm not exactly sure what the problem is/was in your case but it seems like some . Maybe you would have needed to run via bundler with `bundle exec fastlane list_projects`. I'll add this to the README just to be sure.

**Comment by chraibi** [comment]
The problem was simply (I think) that the package `gitlab` could not be found. Just after installing it explicitly everything went well. By the way, many thanks for this nice work. I had a bunch of repos to migrate (over 60) and could just not do it _manually_ :+1:  

Maybe for the readme: I had first to install bundle 

```
gem install bundler
```

**Comment by iv-mexx** [comment]
As you have assumed, `bundle install` should have been enough and installing the `gitlab` gem should have not been necessary. I'm not sure why its sometimes necessary, but running `bundle exec ...` usually works in those cases, so I've added this to the readme just to be sure. 

Also good point about `bundle install`, thanks :+1:
